meta-box/meta-box-template.php

```diff
@@ -22,39 +22,39 @@
 <p>
 	<b>
 		<span class="dashicons dashicons-yes"></span>
-		This post will be available as an Instant Article.
+		This post will be available as Instant Article.
```

Comment:
Maybe:

"This post will be available as Instant Article when shared on Facebook"